Code/Editor/ToolbarManager.h

```diff
@@ -178,6 +184,8 @@ class ToolbarManager
     void UpdateAllowedAreas(QToolBar* toolbar);
     bool IsDirty(const AmazonToolbar& toolbar) const;
 
+    QMenu* GetPlayButtonMenu() const;
```

Comment:
nit: just pointing it out because it threw me, but this is more of a "create" operation than a "get" one IMO